test/user-story-1.spec.js

```diff
@@ -0,0 +1,33 @@
+const { assertEquals } = require("./test-framework");
+const Airport = require("../src/Airport");
+const Plane = require("../src/Plane");
+
+console.log(`________________`);
+console.log(`USER STORY 1 TESTS`);
+console.log(`----------------`);
+console.log(``);
+
```

Comment:
EBI/Goals - more testing for outliers and erroneous inputs. I improved at this in later user stories but earlier ones testing was limited, if I had more time I would have gone back and added more tests to the earlier user stories - eg in user story one additional tests could include:
- Ensure plane landed in airport is a plane object, throw error if not a plane object
- Does plane have valid plane Id
- Testing new instance of airport with default values, does this produce an airport object as expected